### ISSUE TITLE
find_xyz_cut_coords default thresholding fails for binary inputs if integer type

### ISSUE THREAD

**Issue by AlexandreAbraham** [nilearn/nilearn#1194]
By default, a threshold is calculated on non-zero values (which is not exactly what is described in the doc).  If the image is binary, the threshold is 1, which masks everything.

`activation_threshold = fast_abs_percentile(my_map[my_map != 0].ravel(), 80)`

I will fix this in an upcoming PR.

**Comment by GaelVaroquaux** [comment]
> By default, a threshold is calculated on non-zero values (which is not exactly
> what is described in the doc). If the image is binary, the threshold is 1,
> which masks everything.

> activation_threshold = fast_abs_percentile(my_map[my_map != 0].ravel(), 80)

Indeed. The standard solution is to remove 1e-7 to the threshold found.

**Comment by AlexandreAbraham** [comment]
> Indeed. The standard solution is to remove 1e-7 to the threshold found.

I solve it by accepting values equal to the threshold. If failing on integer images is the intended behavior, we should raise an error.

**Comment by GaelVaroquaux** [comment]
> I solve it by accepting values equal to the threshold.

Sounds good.

> If failing on integer images is the intended behavior, we should raise
> an error.

No, it's not.
